MapboxSearch/offline/src/main/java/com/mapbox/search/offline/OfflineSearchCallbackAdapter.kt

```diff
@@ -7,8 +7,10 @@ import com.mapbox.search.base.result.BaseSearchResult
 internal class OfflineSearchCallbackAdapter(private val callback: OfflineSearchCallback) : BaseSearchCallback {
     override fun onResults(results: List<BaseSearchResult>, responseInfo: BaseResponseInfo) {
         callback.onResults(
-            results.map { OfflineSearchResult(it.rawSearchResult) },
-            OfflineResponseInfo(responseInfo.requestOptions.core.mapToOfflineSdkType())
+            results = results
+                .map { OfflineSearchResult(it.rawSearchResult) }
+                .mapNotNull { result -> if (result.name.isBlank()) null else result },
```

Comment:
nit: this could be simplified to
```
results = results
                 .map { OfflineSearchResult(it.rawSearchResult) }
                 .filter { it.name.isNotBlank() }

```

Reply:
Done